docs/event-bus/10-02-troubleshooting-kyma-tracing.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshooting using tracing
+type: Troubleshooting
+---
+
+The tracing functionality available in Kyma can help you to pinpoint the error's root cause and solve the problem. See [this](docs/components/tracing/#overview-overview) document to learn more about tracing.
+
+## No microservice or lambda configured to receive an Event
+
+In this case, an external system sends the Event, but
+a lambda or microservice with an Event trigger does not exist.
+
+As a result, you can see traces for `publish` only, and the trace details show you only the tags for the `event-publish-knative-service`.
```

Comment:
```suggestion
As a result, you can see traces for `publish` only, and the trace details only show you the tags for `event-publish-knative-service`.
```

Reply:
between the subject and the main verb? -> [link](https://github.com/YaaS/REST_API_Documentation_Guidelines/wiki/009-Only-the-Lonely)